changelog/0087-upgrade-to-es-6.yml

```
@@ -0,0 +1,405 @@
+              ```
+
+              This command will display the document counts for both the primary and secondary indices for a given index. If the doc count matches between the two and there are no errors in the reindex logs, then reindexing is complete for that index.
+
+              Please note that the counts may not match perfectly for high frequency indices like case_search, cases, and forms. In such cases, ensure the difference in counts is small (within one hundred) and there are no errors in reindex logs.
+
+              If reindex fails for any index, please refer to the docs [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/REINDEX_PROCESS.md#common-issues-resolutions-during-reindex) for troubleshooting steps.
+
+    4. If doc counts match and there are no errors present in the reindex logs, the reindex for the current index is complete. You can continue reindexing for the next index by repeating steps 7.1-7.3 with `${INDEX_CNAME}` set to 'cases', 'case_search' and so on for all indices.
+
+
+  ## Swap the indices
+
+  Once all the indices have been reindexed, we can swap the indices to start reading and writing to the new index.
+
+  1. Update the index swap settings in the public.yml file.
+
+      ```yaml
+      ES_APPS_INDEX_SWAPPED: True
+      ES_CASE_SEARCH_INDEX_SWAPPED: True
+      ES_CASES_INDEX_SWAPPED: True
+      ES_DOMAINS_INDEX_SWAPPED: True
+      ES_FORMS_INDEX_SWAPPED: True
+      ES_GROUPS_INDEX_SWAPPED: True
+      ES_SMS_INDEX_SWAPPED: True
+      ES_USERS_INDEX_SWAPPED: True
+      ```
+
+  2. Run update config and restart the services to apply the changes.
+
+      ```sh
+      cchq <env> django-manage update_config
+      cchq <env> django-manage restart_services
+      ```
+
+  3. Ensure that HQ is running fine with the new index. You can do it by testing the reports in your hosted CommCare instance. Optional but recommended: You can keep HQ in this state for a few working hours to ensure that the new index is stable.
+
+  4. Turn off multiplexing by setting the following variables to `False` in the public.yml file.
+
+      ```yaml
+      ES_APPS_INDEX_MULTIPLEXED: False
+      ES_CASE_SEARCH_INDEX_MULTIPLEXED: False
+      ES_CASES_INDEX_MULTIPLEXED: False
+      ES_DOMAINS_INDEX_MULTIPLEXED: False
+      ES_FORMS_INDEX_MULTIPLEXED: False
+      ES_GROUPS_INDEX_MULTIPLEXED: False
+      ES_SMS_INDEX_MULTIPLEXED: False
+      ES_USERS_INDEX_MULTIPLEXED: False
+      ```
+
+  5. Run update config and restart the services to apply the changes.
+
+      ```sh
+      cchq <env> django-manage update_config
+      cchq <env> django-manage restart_services
+      ```
+  6. Ensure that HQ is running fine with the new index. You can do it by clicking around HQ and maybe testing a report or two.
+
+  7. Delete old and unused CommCare Indices by running the following commands. The command will ask you confirm the deletion by typing in the index cname. 
```

I assume `index cname` isn't really something self hosters would understand.
```suggestion
  7. Delete old and unused Elasticsearch Indices created by CommCare by running the following commands. The command will ask you confirm the deletion by typing in the index name. 
```